lib/taskana-core/src/main/java/pro/taskana/common/internal/util/FieldAndValuePairTriplet.java

```diff
@@ -0,0 +1,40 @@
+package pro.taskana.common.internal.util;
+
+import java.lang.reflect.Field;
+
+public class FieldAndValuePairTriplet {
+
```

Comment:
Do you mind using our generic triplet class?

Reply:
Sure, changed it